docs/pom.xml

```diff
@@ -93,7 +93,7 @@
                     <mainClass>io.quarkus.docs.generation.AllConfigGenerator</mainClass>
                     <arguments>
                         <argument>${project.version}</argument>
-                        <argument>${project.basedir}/../devtools/bom-descriptor-json/target/extensions.json</argument>
+		        <argument>${project.basedir}/../devtools/platform-descriptor-json/target/classes/quarkus-bom-descriptor/extensions.json</argument>
```

Comment:
Thanks!

Any chance you could keep the indentation as is?

Reply:
Will fix it, thanks. (Damn `vi` on Windows...)